integration-tests/test/native-eth.spec.ts

```diff
@@ -302,4 +303,81 @@ describe('Native ETH Integration Tests', async () => {
     expect(l1BalanceAfter).to.deep.eq(l1BalanceBefore.add(withdrawnAmount))
     expect(l2BalanceAfter).to.deep.eq(amount.sub(withdrawnAmount).sub(fee))
   })
+
+  describe('WETH9 functionality', async () => {
+    let initialBalance: BigNumber
+    const value = 10
+
+    beforeEach(async () => {
+      await fundUser(env.watcher, env.l1Bridge, value, env.l2Wallet.address)
+      initialBalance = await env.l2Wallet.provider.getBalance(
+        env.l2Wallet.address
+      )
+    })
+
+    it('successfully deposits', async () => {
+      const depositTx = await env.ovmEth.deposit({ value, gasPrice: 0 })
+      const receipt = await depositTx.wait()
+
+      expect(
+        await env.l2Wallet.provider.getBalance(env.l2Wallet.address)
+      ).to.equal(initialBalance)
```

Comment:
After a successful deposit why would the balance be the same as the initial balance?

Reply:
Because we're adding `deposit` and `withdraw` to do **nothing**. If you want to "deposit" (wrap) your ETH into WETH, you actually don't need to do anything, since ETH is already an ERC20 in L2. This is just being added to be backwards compatible with people's contracts that assume that you have to call `deposit` to wrap your ETH to be ERC20-compatible.